mu4/importexport/internal/guitarpro/importgtp-gp6.cpp

```diff
@@ -1131,10 +1131,6 @@ Fraction GuitarPro6::readBeats(QString beats, GPPartInfo* partInfo, Measure* mea
                                     if (!currentProperty.firstChild().nodeName().compare("Enable")) {
                                         addPalmMute(note);
                                     }
-                                } else if (argument == "Tapped") {
-                                    if (!currentProperty.firstChild().nodeName().compare("Enable")) {
-                                        addTap(note);
-                                    }
```

Comment:
why this removed?

Reply:
Because there is the exact same case at line 1066: it is a duplicated condition.